UI/EncyclopediaDetailPanel.cpp

```diff
@@ -1719,30 +1675,26 @@ namespace {
         auto unlocked_by_techs = TechsThatUnlockItem(UnlockableItem(UnlockableItemType::UIT_BUILDING, item_name));
         if (!unlocked_by_techs.empty()) {
             detailed_description += "\n\n" + UserString("ENC_UNLOCKED_BY");
-            for (const std::string& tech_name : unlocked_by_techs)
-                detailed_description += LinkTaggedPresetText(VarText::TECH_TAG, tech_name, UserString(tech_name)) + "  ";
+            detailed_description.append(LinkList(unlocked_by_techs));
             detailed_description += "\n\n";
         }
 
         auto unlocked_by_policies = PoliciesThatUnlockItem(UnlockableItem(UnlockableItemType::UIT_BUILDING, item_name));
         if (!unlocked_by_policies.empty()) {
             detailed_description += "\n\n" + UserString("ENC_UNLOCKED_BY");
-            for (const auto& policy_name : unlocked_by_policies)
-                detailed_description += LinkTaggedPresetText(VarText::POLICY_TAG, policy_name, UserString(policy_name)) + "  ";
+            detailed_description.append(LinkList(unlocked_by_policies));
         }
 
         // species that like / dislike building type
         auto species_that_like = GetSpeciesManager().SpeciesThatLike(item_name);
         auto species_that_dislike = GetSpeciesManager().SpeciesThatDislike(item_name);
         if (!species_that_like.empty()) {
             detailed_description += "\n\n" + UserString("SPECIES_THAT_LIKE");
-            for (auto& name : species_that_like)
-                detailed_description += LinkTaggedPresetText(VarText::SPECIES_TAG, name, UserString(name)) + " ";
+            detailed_description.append(LinkList(species_that_like));
         }
         if (!species_that_dislike.empty()) {
             detailed_description += "\n\n" + UserString("SPECIES_THAT_DISLIKE");
-            for (auto& name : species_that_dislike)
-                detailed_description += LinkTaggedPresetText(VarText::SPECIES_TAG, name, UserString(name)) + " ";
+            detailed_description.append(LinkList(species_that_dislike));
```

Comment:
with this commit it looks more linked up then it was before:

![Bildschirmfoto 2022-01-21 um 23 06 38](https://user-images.githubusercontent.com/37298400/150607664-ad998040-42f9-4896-96c3-e51834c67c97.png)
 
but that last one still missing.